ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldLRspec.json

```diff
@@ -1259,5 +1259,78 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "responseClaimAdmitPartOfClaimEmploymentDeclarationLabel",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "[RESPONDENTSOLICITORONE]"
+        ],
+        "CRUD": "CRU"
```

Comment:
Does a label need CU permissions? Actually, does it even need R permissions to be shown?

Reply:
removed json block. and it was displayed normally. so no authentication needed for this.